devdox/app/main.py

```
@@ -15,21 +16,52 @@
-
+async def init_mongo(mongo_settings:MongoConfig | None):
+
+    if mongo_settings:
+        mongo_uri = build_uri(mongo_conf=mongo_settings)
+        mongo_client, mongo_db = await init_via_uri(mongo_uri)
+        
+        # health check to fail fast
+        try:
+            await mongo_db.command("ping")
+        except Exception:
+            # clean up before re-raising so FastAPI doesn’t keep a half-open client
```

Ruff (0.14.5)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)